stock_cycle_count/models/stock_location.py

```diff
@@ -0,0 +1,119 @@
+# Copyright 2017-18 ForgeFlow S.L.
+#   (http://www.forgeflow.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+import logging
+from datetime import datetime
+
+from odoo import fields, models, tools
+from odoo.tools import DEFAULT_SERVER_DATETIME_FORMAT
+
+_logger = logging.getLogger(__name__)
+
+try:
+    from statistics import mean
+
+    STATS_PATH = tools.find_in_path("statistics")
+except (ImportError, IOError) as err:
+    _logger.debug(err)
+
+
+class StockLocation(models.Model):
+    _inherit = "stock.location"
+
+    def _compute_loc_accuracy(self):
+        for rec in self:
+            history = self.env["stock.inventory"].search(
```

Comment:
IMHO this should be refined to improve performances.

Reply:
You didn't solve this. You should put the search before the loop with 'location_ids', 'in', self.ids.

You can then add here history(_ids).filtered_domain([("location_ids", "in", rec.id)])

Reply:
Done